packages/cli/src/commands/open.ts

```diff
@@ -0,0 +1,325 @@
+    'DIR'
+  )}    Path to the global ${'`.vercel`'} directory
+
+  ${chalk.dim('Examples:')}
+
+  ${chalk.gray('–')} View all options
+
+    ${chalk.cyan(`$ ${getPkgName()} open`)}
+
+  ${chalk.gray('–')} Open the dashboard for the current project
+
+    ${chalk.cyan(`$ ${getPkgName()} open dash`)}
+
+  ${chalk.gray('–')} Open the latest preview deployment URL
+
+    ${chalk.cyan(`$ ${getPkgName()} open latest`)}
+
+  ${chalk.gray('–')} Open the latest production deployment URL
+
+    ${chalk.cyan(`$ ${getPkgName()} open latest --prod`)}
+
+  ${chalk.gray('–')} Open the dashboard for the latest preview deployment
+
+    ${chalk.cyan(`$ ${getPkgName()} open dash latest`)}
+
+  ${chalk.gray('–')} Open the dashboard for the latest production deployment
+
+    ${chalk.cyan(`$ ${getPkgName()} open dash latest --prod`)}
+
+  ${chalk.gray('–')} Open a specific deployment URL
+  
+    ${chalk.cyan(`$ ${getPkgName()} open [url]`)}
+
+  ${chalk.gray('–')} Open the dashboard for a specific deployment URL
+  
+    ${chalk.cyan(`$ ${getPkgName()} open dash [url]`)}
+`);
+};
+
+export default async function open(
+  client: Client,
+  test: Boolean
+): Promise<number> {
+  const { output } = client;
+  let argv;
+  let subcommand: string | string[];
+  let narrow: string | string[];
+
+  try {
+    argv = getArgs(client.argv.slice(2), {
+      '--confirm': Boolean,
+      '--prod': Boolean,
+    });
+  } catch (error) {
+    handleError(error);
+    return 1;
+  }
+
+  argv._ = argv._.slice(1);
+  subcommand = argv._[0];
+  narrow = argv._[1];
+
+  if (argv['--help']) {
+    help();
+    return 2;
+  }
+
+  const confirm = argv['--confirm'] || false;
+  const prod = argv['--prod'] || false;
+
+  let scope = null;
+
+  try {
+    scope = await getScope(client);
+  } catch (err) {
+    if (err.code === 'NOT_AUTHORIZED' || err.code === 'TEAM_DELETED') {
+      output.error(err.message);
+      return 1;
+    }
+
+    throw err;
+  }
+
+  const { team, contextName } = scope;
+
+  let paths = [process.cwd()];
+
+  const validate = await validatePaths(client, paths);
+  if (!validate.valid) {
+    return validate.exitCode;
+  }
+  const { path } = validate;
+
+  const linkedProject = await ensureLink('open', client, path, confirm);
+  if (typeof linkedProject === 'number') {
+    return linkedProject;
+  }
+
+  const { project, org } = linkedProject;
+  client.config.currentTeam = org.type === 'team' ? org.id : undefined;
+
+  const choice = await getChoice(
+    subcommand,
+    narrow,
+    contextName,
+    client,
+    project,
+    org,
+    team,
+    prod
+  );
+  if (typeof choice === 'number') {
+    return choice;
+  }
+
+  if (choice === 'not_found') {
+    output.log(
+      `No deployments found. Run ${chalk.cyan(
+        getCommandName('deploy')
+      )} to create a deployment.`
+    );
+    return 1;
+  }
+  if (choice === '') {
+    // User aborted
+    return 0;
+  }
+
+  if (!test) execa('open', [choice]);
```

Comment:
This will only work on natively macOS, I believe. Should we use a package that will make sure this is cross-platform?

I've not used a package like this in the past. I found this one that looks alright: https://www.npmjs.com/package/open

Reply:
Wow, thank you for catching this.